### DISCUSSION TITLE
charging limit only works on 80%，not working on 90% G14(2024)

### DISCUSSION THREAD

**Discussion by vansonffff** [seerge/g-helper#2172] [Q&A] [answered]
already delete MY ASUS
why charging limit only works on 80%？？
model：G14 (2024)

**Comment by seerge** [comment, marked as answer]
https://github.com/seerge/g-helper/wiki/FAQ#battery-charge-limiter-is-not-working

**Reply by vansonffff** [reply]
“ For some devices not every charge limit % may be working. Some models support only 80% limit, so try to set standard 80% to be sure”
Maybe it‘s the answer.
Thank you！！！
